contracts/utils/GSNBouncerBase.sol

```diff
@@ -0,0 +1,92 @@
+pragma solidity ^0.5.0;
```

Comment:
In case you didn't modify any of the GSN/Relay contracts I would suggest to import them directly from the vendors folder so it reduce the friction of maintain and update them. You should also use fixed version for those packages to prevent any kind of unexpected upgrade on them.

Reply:
it is a better practice but I did it on purpose. The reason is bc I will need to copy those SC to Leth Go library anyway so in this way I can be sure I know all the SCs that I need and if they work. Otherwise if I require them using npm here then I loose this ability.

UPDATE: After this morning we decided not to implement GSN as of the moment to Smart Vault Go project therefore I can require the contracts using npm paths here as I won't have to manually copy them to the Smart Vault Go project.